documentation/decisions/adr/2023-12-15-deliverable-reporting-strategy.md

```diff
@@ -0,0 +1,235 @@
+- using that column to indicate the 30k deliverable that an issue is assigned to
+
+- **Positive outcomes**
+  - We can filter, group, and sort issues by deliverable in GitHub projects and GitHub's automated insight reporting.
+  - The engineering team can continue to use GitHub milestones to organize issues into units of work that are smaller than a 30k deliverable.
+- **Negative outcomes**
+  - We won't be able to filter for issues assigned to a given 30k deliverable within the GitHub repository.
+  - We'll have to make sure the list of options in the deliverable column is consistent across GitHub projects in order to join issues from different projects correctly in our custom reporting.
+  - We'll have to update the current logic in our custom reporting so that issues are joined to their parent deliverable using the value of this column.
+
+> [!NOTE]
+> If the team finds a consistent need to filter the list of issues by deliverable within the repository, we may revisit this decision and choose to use ***both*** a label and a deliverable column to assign an issue to a 30k deliverable. We've decided **not** to use both options for the time being to avoid having multiple (potentially conflicting) ways of assigning an issue to a given deliverable.
+
+### Limiting the 30k deliverables included in reporting
+
+We've decided to use the status of the deliverable in the product roadmap board to determine when deliverables are included or excluded from reporting.
+
+- **Positive outcomes**
+  - Keeps our reporting strategy closed aligned with how we're monitoring our delivery progress.
+  - Avoids creating *another* field or label that needs to be maintained separately.
+- **Negative outcomes**
+  - Tightly couples reporting logic with delivery management, which may introduce challenges if there are instances in which we want to report on some but not all deliverables in a given status.
+  - If stakeholders aren't familiar with the relationship between deliverable status and reporting, it may be harder for them to understand why certain deliverables appear in a given report but others do not.
+
+## Evaluation - Assigning issues to deliverables <!-- OPTIONAL -->
+
+### Milestone
+
+This option involves creating a milestone for each 30k deliverable and assigning issues to that milestone if they are required for delivery of that 30k.
+
+> [!NOTE]
+> This is a slightly different use of milestones than the project currently supports. Right now, the engineering team can create multiple milestones for a given 30k deliverable and reporting is driven by tagging the parent 30k deliverable in the description of the milestone. This new approach would require a *single* milestone per 30k deliverable and all tickets associated with that deliverable would need to be added to the milestone to be included in reporting.
+
+> [!TIP]
+> **Bottom line:** This is the best option if we:
```

Comment:
I really like the "This is the best option if you want X, Y, Z" framing! Nice innovation.